packages/logs/src/domain/logsSessionManager.ts

```diff
@@ -24,11 +25,12 @@ export function startLogsSessionManager(configuration: LogsConfiguration): LogsS
     computeSessionState(configuration, rawTrackingType)
   )
   return {
-    findTrackedSession: (startTime) => {
+    findTrackedSession: (startTime = relativeNow()) => {
       const session = sessionManager.findSession(startTime, isActiveOrInactiveSessionPredicate)
       return session && session.trackingType === LoggerTrackingType.TRACKED
         ? {
             id: session.id,
+            active: (session.endTime || Infinity) > startTime,
```

Comment:
I've exposed [isActiveAt](https://github.com/DataDog/browser-sdk/blob/4a5df2185135494bd4f59c3fa85446ac90186845/packages/logs/src/domain/logsSessionManager.ts#L32). Let me know what you think.

Reply:
Counter-proposal: what about adding an option to the `findTrackedSession` method to return only active session by default but allow to return expire sessions?
something like:
```
findTrackedSession(startTime, { returnExpired: true })
```

so by default it would have the same behaviour than RUM but when we need to also check for expired sessions it would be explicit.

Reply:
Hum, it would mean to have 2 findTrackedSession() calls in the logsAssembly:

- One for the check 
```typescript
const session = sessionManager.findTrackedSession(startTime, { returnExpired: true })
if (!session) {
    return
}
```
- Another to set the session id
```typescript
const log = combine(
{
session_id: sessionManager.findTrackedSession(startTime)?.id,
...
}
```
If that sounds good for you, it's good for me :)

Reply:
Hum, that does not really say what it is doing though 🤔

It would be nice to be be able to express:
```js
{
  session_id: !session.isExpired : session.id : undefined,
  ...
}
```

But then this property would be computed from the `startTime` which is what I found confusing originally 😕

I like the idea of not impacting other places where we are checking for active tracked session and being explicit about what we are doing in the logs assembly.

I'll let you choose the approach that seems best with your understanding 😉

Reply:
Okay, so I chose to have both { returnExpired: true } option and the isActiveAt() method to be more explicit. So basically in the assembly we have

```typescript
const session = sessionManager.findTrackedSession(startTime, { returnExpired: true })
if (!session) {
  return
}
const log = combine(
{
  session_id: session.isActiveAt(startTime) ? session.id : undefined,
  ...
}
```
Let me know what you think